collect/collect.go

```diff
@@ -707,6 +712,9 @@ func (i *InMemCollector) send(trace *types.Trace, reason string) {
 		if i.Config.GetAddRuleReasonToTrace() {
 			sp.Data["meta.refinery.reason"] = reason
 		}
+		if i.Config.GetStressReliefConfig().Mode != "" {
+			sp.Event.Data["meta.refinery.local_hostname"] = i.hostname
```

Comment:
It looks like this function is already setting `local_hostname` whenever `i.hostname` is set on line 731.

Reply:
Okay, changed all instances to adopt that pattern.